modules/dnn/src/layers/matmul_layer.cpp

```diff
@@ -135,6 +198,36 @@ class MatMulLayerImpl CV_FINAL : public MatMulLayer {
         const auto *a = A.ptr<const float>();
         auto *y = Y.ptr<float>();
         std::memset(y, 0, Y.total() * sizeof(float));
```

Comment:
Seems `memset` can be moved to else if there is no bias:
```cpp
if ((inputs.size() + blobs.size()) == 3) {
} else {
   std::memset(y, 0, Y.total() * sizeof(float));
}
```

Reply:
Resolved